src/js/components/locations-configuration/LocationDetails.jsx

```diff
@@ -207,41 +265,60 @@ class LocationDetails extends Component {
   render() {
     return (
       <div className="configuration-wizard-content flex-column">
+        <AddOrganizationModal
+          isOpen={this.state.showOrganizationModal}
+          onClose={() => this.setState({ showOrganizationModal: false })}
+          onResponse={this.setOrganization}
```

Comment:
when you move this, you will be able to use here `form.mutators.changeOrganization`